components/strava/actions/list-activities/list-activities.mjs

```diff
@@ -0,0 +1,40 @@
+import strava from "../../strava.app.js";
+
+export default {
+  name: "Get Activity List",
+  description: "Returns the activities of an athlete for a specific identifier. See `https://developers.strava.com/docs/reference/`",
```

Comment:
Could you please use []() to ref the docs?

Reply:
done